ROLES.md

```diff
@@ -259,36 +241,14 @@ Administrators are responsible for the bureaucratic aspects of the project.
 
 ### Responsibilities and privileges
 
-* Manage the Istio GitHub repo, including granting membership, controlling repo read/write permissions,
-and team membership
-
-* Manage the Istio Slack workspace
-
-* Manage the Istio Google group forums
-
-* Manage the Google Search Console settings for istio.io
-
-## Vendor
-
-Vendors contribute extensions to the Istio project in the form of new adapters, translated documentation, interesting examples, etc.
-
-### Requirements
-
-* <a href="#sponsor">Sponsored</a> by a member
-
-### Responsibilities and privileges
-
-* Each vendor receives access to a subdirectory in the contrib repo, and each directory has got a distinct OWNERS file granting
-approver permissions to the vendor.
+* Manage a variety of Istio-project infrastruture aa described in [ADMINS-FOR-ISTIO](./ADMINS-FOR-ISTIO.md)
 
 ## Sponsor
 
-A sponsor is a member that wishes to vouch for another user to become a member.
+A sponsor is a member that wishes to vouch for another user to support a user being assigned a specific role.
```

Comment:
Can sponsors vouch for another user to have a greater role than they have (e.g., can a non-maintainer vouch for someone to become a maintainer)?